### DISCUSSION TITLE
[Feature Request] Support Solana and SPL tokens on Ledger Live

### DISCUSSION THREAD

**Discussion by NicholasJonesCode** [LedgerHQ/ledger-live-desktop#4005] [Coin/Token request]
Hey Ledger Team, 

I am wondering when the Solana chain is going to be supported on the Ledger Live app? Solana has become popular very quickly, with its lightning fast transactions and extremely low network fees, along with ERC-20-like tokens with its SPL tokens. Solana even has native USDC and USDT, along with popular wrapped tokens like ETH and XRP

I think all-in-all it really deserves to have an interface on the Ledger app. More than dogecoin at least :) I use Solana and several SPL tokens a lot and it would really make things much easier for me. Part of the reason i bought a Ledger wallet initially was for my Solana tokens.

Please let me know what your team's thoughts are, thanks!

**Comment by GentlemansKiller** [comment]
Yes Please!

**Comment by RabbiShekelberg** [comment]
Shalom my fellow programmers. I strongly approve this message.

**Comment by TexasRedHuncho** [comment]
Please do this, It would be really helpful.

**Comment by akuz** [comment]
Yes, please!

**Comment by TimmyBoi155** [comment]
I need this!!

**Comment by kaifri** [comment]
I need this aswell. please

**Comment by sunlee-newyork** [comment]
Please add Solana to Ledger Live!

**Comment by rsaulo** [comment]
Please!!!! 
It is mandatory support for Solana SPL tokens in Ledger Live!!!

**Comment by ElMalek2015** [comment]
Yes please. ????

**Comment by CruiseMan** [comment]
I DON'T UNDERSTAND WHY LEDGER TEAM IS SO IGNORANT TO EVEN COMMENT ON THIS

**Reply by batscs** [reply]
From what I understand it's not in the hands of the Ledger Team, the Solana Devs need to write the plugin for the Ledger. (Take this with a grain of salt, no idea how true this is, just read it somewhere online lol)

Edit: yeah in hindsight this is probably wrong so take this with a huge gigantic grain of salt bricks

**Reply by johnpc** [reply]
I find it difficult to believe Solana devs would need to get involved. Especially just to add balances/withdraw/deposit to ledger live, which is all most people want.

**Reply by CruiseMan** [reply]
Trust Wallet multicoin wallet  did it and it's open source https://github.com/trustwallet/wallet-core, Ledger is just blind and incompetent instead adding SOL they do shit

**Comment by CruiseMan** [comment]
> From what I understand it's not in the hands of the Ledger Team, the Solana Devs need to write the plugin for the Ledger. (Take this with a grain of salt, no idea how true this is, just read it somewhere online lol)

They could add native support for Solana

**Reply by plk** [reply]
Would also support this. Solana is basically the number three (real) coin after Bitcoin and Ethereum now and should be a priority above all other integrations.

**Comment by milk-duds-pro** [comment]
Yeah, this would be really nice actually. As it is i have to connect to a website like sollet or something just to send any tokens

**Comment by johnpc** [comment]
https://twitter.com/Ledger/status/1469240129319579648

It's happening
